modules/installing-microshift-rpm.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+:_content-type: PROCEDURE
+[id="installing-microshift-from-rpm-package_{context}"]
+= Installing MicroShift from an RPM package 
+
+Use the following procedure to install MicroShift from an RPM package. 
+
+.Prerequisites 
+
+* The xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[system requirements for installing MicroShift] have been met. 
+* You have completed the steps at xref:../microshift_install/microshift-install-rpm.adoc#preparing-install-microshift-from-rpm-package_microshift-install-rpm[Preparing your system for installing MicroShift from an RPM package].
+
+.Procedure
+
+. Enable the MicroShift repositories by entering the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos \
+    --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms \
+    --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+----
+
+. Install MicroShift by entering the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y microshift
+----
+
+. Download your installation link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat Hybrid Cloud Console]. This pull secret allows you to authenticate with the container registries that serve the container images used by MicroShift.
+
+. Copy the pull secret to your {op-system} machine the `/etc/cio/openshift-pull-secret` folder of your {op-system} machine. 
+
+. Make the root user the owner of the `/etc/crio/openshift-pull-secret` folder by entering the following command: 
++
+[source,terminal]
+----
+$ sudo chown root:root /etc/crio/openshift-pull-secret
+----
+
+. Make the `/etc/crio/openshift-pull-secret` folder readable and writeable by the root user only by entering the following command:
++
+[source,terminal]
+----
+$ sudo chmod 600 /etc/crio/openshift-pull-secret
+----
+
+. If your {op-system} machine uses `firewalld`, configure the mandatory rules by entering the following commands: 
```

Comment:
`firewalld` is the default, but there's a small chance users are using some other firewall (e.g. just `iptables`). If we feel we want to hedge for that case, we may want to change this to something like:

"If your  {op-system} machine has a firewall enabled, you need to configure a few mandatory firewall rules. For `firewalld` enter the following commands:"